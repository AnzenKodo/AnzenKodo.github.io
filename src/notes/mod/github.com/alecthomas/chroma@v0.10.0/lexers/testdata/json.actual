{
	"list": [1, 2, 3],
	"map": {
		"key": "value",
	},
	// A comment
	"number": 1.23,
	"string": "string",
	"positive": true,
	"negative": false
}