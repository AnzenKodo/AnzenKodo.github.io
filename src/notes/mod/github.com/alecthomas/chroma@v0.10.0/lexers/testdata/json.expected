[
  {"type":"Punctuation","value":"{"},
  {"type":"Text","value":"\n\t"},
  {"type":"NameTag","value":"\"list\""},
  {"type":"Punctuation","value":":"},
  {"type":"Text","value":" "},
  {"type":"Punctuation","value":"["},
  {"type":"LiteralNumberInteger","value":"1"},
  {"type":"Punctuation","value":","},
  {"type":"Text","value":" "},
  {"type":"LiteralNumberInteger","value":"2"},
  {"type":"Punctuation","value":","},
  {"type":"Text","value":" "},
  {"type":"LiteralNumberInteger","value":"3"},
  {"type":"Punctuation","value":"],"},
  {"type":"Text","value":"\n\t"},
  {"type":"NameTag","value":"\"map\""},
  {"type":"Punctuation","value":":"},
  {"type":"Text","value":" "},
  {"type":"Punctuation","value":"{"},
  {"type":"Text","value":"\n\t\t"},
  {"type":"NameTag","value":"\"key\""},
  {"type":"Punctuation","value":":"},
  {"type":"Text","value":" "},
  {"type":"LiteralStringDouble","value":"\"value\""},
  {"type":"Punctuation","value":","},
  {"type":"Text","value":"\n\t"},
  {"type":"Punctuation","value":"},"},
  {"type":"Text","value":"\n\t"},
  {"type":"CommentSingle","value":"// A comment\n"},
  {"type":"Text","value":"\t"},
  {"type":"NameTag","value":"\"number\""},
  {"type":"Punctuation","value":":"},
  {"type":"Text","value":" "},
  {"type":"LiteralNumberFloat","value":"1.23"},
  {"type":"Punctuation","value":","},
  {"type":"Text","value":"\n\t"},
  {"type":"NameTag","value":"\"string\""},
  {"type":"Punctuation","value":":"},
  {"type":"Text","value":" "},
  {"type":"LiteralStringDouble","value":"\"string\""},
  {"type":"Punctuation","value":","},
  {"type":"Text","value":"\n\t"},
  {"type":"NameTag","value":"\"positive\""},
  {"type":"Punctuation","value":":"},
  {"type":"Text","value":" "},
  {"type":"KeywordConstant","value":"true"},
  {"type":"Punctuation","value":","},
  {"type":"Text","value":"\n\t"},
  {"type":"NameTag","value":"\"negative\""},
  {"type":"Punctuation","value":":"},
  {"type":"Text","value":" "},
  {"type":"KeywordConstant","value":"false"},
  {"type":"Text","value":"\n"},
  {"type":"Punctuation","value":"}"}
]
